data-management/neo4j-service.js

```diff
@@ -631,6 +631,24 @@ class Neo4jService {
         return result[0];
     }
 
+    async deleteExpiredTokenUUIDs() {

```

Comment:
> Let's simplify this and store the expiration time as a timestamp instead of the time until expiration. Then we can check for expired tokens with a query similar to this without relying on events:
> 
> MATCH (token:Token) WHERE toInteger(token.token_expiration) < toInteger(timestamp()) WITH COLLECT(token.uuid) AS token_uuids DETACH DELETE token RETURN token_uuids

Reply:
Simplified the query for the update.